scripts/build-windows-k8s.sh

```diff
@@ -87,11 +87,28 @@ k8s_17_cherry_pick() {
         git cherry-pick 51fab673e1..72b9c8f519
 }
 
+k8s_18_cherry_pick() {
+	# 72b9c8f519 Add start time for root container spec
+	# b7c4184821 Fix windows docker stats cpu units issue
+	# 51fab673e1 Merge pull request #3 from JiangtianLi/release-1.7
+	# 45ba7bb0fb Implement metrics for Windows Containers
+	# 76b94898ec Use dns policy to determine setting DNS servers on the correct NIC in Windows container
+	# 74a2f37447 Fix network config due to the split of start POD sandbox and start container from 1.7.0
+	# 5fc0a5e4a2 Workaround for Outbound Internet traffic in Azure Kubernetes (*) Connect a Nat Network to the container (Second adapter) (*) Modify the route so that internet traffic goes via Nat network, and POD traffic goes over the CONTAINER_NETWORK (*) Modify getContainerIP to return the IP corresponding to POD network, and ignore Nat Network (*) DNS Fix for ACS Kubernetes in Windows
+	# adeb88d774 Use adapter vEthernet (HNSTransparent) on Windows host network to find node IP
+	# 02549d6647 Merge pull request #50914 from shyamjvs/add-logging-to-logdump
+
+	git cherry-pick 02549d6647..45ba7bb0fb
```

Comment:
Please change commit after I cherry-pick and push. Also please use git cherry-pick a^..b, which is easy to read.